### ISSUE TITLE
Ratings and tags management

### ISSUE THREAD

**Issue by ghost** [beetbox/beets#122]
This issue was automatically migrated from Google Code.
_Original author: michele....@gmail.com (December 12, 2012 00:20:02)_
_Original issue: https://github.com/google-code-export/beets/issues/471_

**Comment by pirish** [comment]
This would be a killer feature for me as well. The biggest issue keeping me from abandoning iTunes is ratings and smart playlists. I finally gave up on iPhones because after 7 hardware revisions ( 5 when I ditched) and 8 major software releases you still can't create a new smart playlist without syncing to the desktop.  This doesn't seem to be that difficult and issue to solve pragmatically.  Is this being worked on or has anyone even claimed it?

**Comment by sampsyo** [comment]
No, I don't think anyone's currently hacking on it. Green field!

**Comment by alphapapa** [comment]
@pirish I'm not exactly sure how iTunes' smart playlists work, but have you checked Clementine?  It has smart playlists and dynamic playlists which work very well.
